Find a functionally equivalent code:android.app.Notification.Style.addExtras:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -5,8 +5,9 @@
     if (mBigContentTitle != null) {
         extras.putCharSequence(EXTRA_TITLE_BIG, mBigContentTitle);
     }
+    extras.putString(EXTRA_TEMPLATE, this.getClass().getName());
 }

